awx/main/management/commands/get_custom_venv_associations.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2021 Ansible, Inc.
+# All Rights Reserved
+
+from django.core.management.base import BaseCommand
+from awx.main.models import Organization, InventorySource, JobTemplate, Project
+
+
+class Command(BaseCommand):
+    """Returns the pip freeze from the path passed in the argument"""
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'path',
+            type=str,
+            nargs=1,
+            default='',
+            help='run this with a path to a virtual environment as an argument to see the pip freeze data',
+        )
+
+    def handle(self, *args, **options):
+        # look organiztions and unified job templates (which include JTs, workflows, and Inventory updates)
+        super(Command, self).__init__()
+        results = {}
+        if options.get('path'):
+            # sanity check here - is path in list
+            path = options.get('path')[0]
```

Comment:
I would add validation that this is in list of custom venvs, otherwise we can get tripped up --especially with trailing slash

```
# works as expected
# awx-manage get_custom_venv_associations /tmp/elijahfoo
{'Organizations': [], 'Job Templates': [{'name': 'Demo Job Template', 'id': 7}], 'Projects': [], 'Inventory Sources': []}

# does not
# awx-manage get_custom_venv_associations /tmp/elijahfoo/
{'Organizations': [], 'Job Templates': [], 'Projects': [], 'Inventory Sources': []}
```